frontend/src/components/dashboard/campaigns/announcement-modal/GraphicAnnouncementModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import styles from './GraphicAnnouncementModal.module.scss'
+import AnnouncementModalOptions from './AnnouncementModalOptions'
+import { AnnouncementModalProps } from './AnnouncementModal'
+
+const GraphicAnnouncementModal = ({
+  title,
+  mediaUrl,
+  subtext,
+  primaryButtonUrl,
+  primaryButtonText,
+  secondaryButtonUrl,
+  secondaryButtonText,
+  handleReadMoreClicked,
+}: AnnouncementModalProps) => {
+  return (
+    <>
+      <img className={styles.graphic} src={mediaUrl} alt="Modal graphic"></img>
```

Comment:
```suggestion
      { mediaUrl && <img className={styles.graphic} src={mediaUrl} alt="Modal graphic"></img> }
```

Text only announcement should not display images.

![image](https://user-images.githubusercontent.com/3666479/108296876-4f756b00-71d5-11eb-92f1-2c3ede415568.png)